src/QMCWaveFunctions/tests/test_dirac_det.cpp

```diff
@@ -168,6 +175,20 @@ void FakeSPO::evaluate(const ParticleSet& P,
     psi[i] = a(iat, i);
   }
 }
+
+void FakeSPO::evaluate(const ParticleSet& P,
```

Comment:
Is this function  been called?

Reply:
It's not called.  This is a side effect of having this function pure virtual in the base class, and therefore having to have an instantiation for each derived class.  This can go away when I implement the suggestion you made previously.